member.py

```diff
@@ -190,6 +189,7 @@ def get_add_gig_band_list(cls, req, the_member_key):
 
     @classmethod
     def get_forums(cls, req, the_member_key):
+        import forum
```

Comment:
This bit is the one thing I haven't tested that I'm pretty sure is safe but not certain. For some reason running the unit tests (i.e. with a __main__ entry point) complains about circular dependencies, when loading the app from dev_appserver or the production App Engine does not.